config/routes.rb

```diff
@@ -6,6 +6,8 @@
   namespace :employees do
     resources :kudos
     resources :rewards, only: [:index, :show]
+     resources :orders, only: [:create, :new], module: :rewards
```

Comment:
there is also no need for resources `orders/new`, as you don't use it anywhere

Reply:
same as up